src/Figshare.jl

```diff
@@ -0,0 +1,74 @@
+struct Figshare <: DataRepo
+end
+
+base_url(::Figshare) = "https://api.figshare.com/v2/articles"
+
+function description(repo::Figshare, mainpage)
+    desc = mainpage["description"]
+    authors = [value["full_name"] for value in mainpage["authors"]]
+    author = format_authors(authors)
+    license = mainpage["license"]["name"] * " (" *mainpage["license"]["url"] * ")"
+    date = mainpage["published_date"]
+    paper = mainpage["citation"]
+    
+    escape_multiline_string("""
+    Author: $(author)
+    License: $(license)
+    Date: $(date)
+
+    $(desc)
+
+    Please cite this paper:
+    $(paper)
+    if you use this in your research.
+    """, "\$")
+end
+
+function get_urls(repo::Figshare, page)
+    urls = []
+    for i in page["files"]
+        push!(urls, i["download_url"])
+    end
+    urls
+end
+
+function get_checksums(repo::Figshare, page)
+    checksums = []
+    for i in page["files"]
+        push!(checksums, i["computed_md5"])
+    end
+    checksums
```

Comment:
Need to do the same thing as for the other that is MD5,
give a message telling user it is MD5 and setting it so that `(md5, "1234A12")` is produced in the output.
Could abstract out a common unction for that.

Reply:
Wait how did it pass tests locally if it was not using MD5 to check MD5s?

Reply:
Because we don't have Integration Tests for others using MD5.